index.html

```diff
@@ -43,5 +43,162 @@ <h2 class="heading-2">I am Software Developer</h2>
+                      <ul class="stories-lang-list">
+                          <li class="stories-list-article">html</li>
+                          <li class="stories-list-article">bootstrap</li>
+                          <li class="stories-list-article">Ruby</li>
+                      </ul>
+                  </div>
+            </div>
+        <button type="submit" class="button-articles">See Project</button>
+    
+            <div class="article-1">
+                <h4 class="heading-article"> Profesional Art Printing Data</h4>
+                  <p class="heading-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                
+                  <div class="Stories-lang-article-1">
+                      <ul class="stories-lang-list">
+                          <li class="stories-list-article">html</li>
+                          <li class="stories-list-article">bootstrap</li>
+                          <li class="stories-list-article">Ruby</li>
+                      </ul>
+                  </div>
+            </div>
+        <button type="submit" class="button-articles">See Project</button>
+    
+            <div class="article-1">
+                <h4 class="heading-article"> Profesional Art Printing Data</h4>
+                  <p class="heading-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                
+                  <div class="Stories-lang-article-1">
+                      <ul class="stories-lang-list">
+                          <li class="stories-list-article">html</li>
+                          <li class="stories-list-article">bootstrap</li>
+                          <li class="stories-list-article">Ruby</li>
+                      </ul>
+                  </div>
+            </div>
+        <button type="submit" class="button-articles">See Project</button>
+    
+            <div class="article-1">
+                <h4 class="heading-article"> Profesional Art Printing Data</h4>
+                  <p class="heading-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                
+                  <div class="Stories-lang-article-1">
+                      <ul class="stories-lang-list">
+                          <li class="stories-list-article">html</li>
+                          <li class="stories-list-article">bootstrap</li>
+                          <li class="stories-list-article">Ruby</li>
+                      </ul>
+                  </div>
+            </div>
+            
+        <button type="submit" class="button-articles">See Project</button>
+    
+          </div> 
+        </section>
+    
+        <!-- About me -->
+
+        <section class="about-me-section">
+            <div class="about-title">
+                <h1>About me</h1>
```

Comment:
- [ ] Let's change this `h1` tag to another `hx` tag, between `h2...h6`. This is because the h1 element is one of the tags that common web search engines (like google) look at when indexing a web page. Having multiple `h1` tags doesn't allow these engines to properly classify your page.